.github/workflows/snapshot_deploy.yml

```diff
@@ -12,28 +13,16 @@ defaults:
   run:
     shell: bash
 
+permissions:
+  attestations: write
+  id-token: write
+  contents: read
+
 jobs:
   deploy-snapshot:
-    runs-on: ubuntu-latest
-    name: Deploy Snapshot artifacts
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: actions/setup-java@v4
-        with:
-          distribution: 'temurin'
-          java-version: 17
-          cache: 'maven'
-          server-id: 'ossrh'
-          server-username: MAVEN_USERNAME
-          server-password: MAVEN_PASSWORD
-          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
-          gpg-passphrase: MAVEN_GPG_PASSPHRASE
-
-      - name: Deploy ${{steps.metadata.outputs.next-version}}
-        run: |
-          mvn -B clean deploy -DskipTests -DperformRelease=true -Drelease
-        env:
-            MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
-            MAVEN_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
-            MAVEN_GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
\ No newline at end of file
+    name: Deploy Snapshots
+    uses: quarkiverse/.github/.github/workflows/perform-release.yml@main
+    secrets: inherit
+    with:
+      ref: main-lts
+      version: 3.0.0-SNAPSHOT
```

Comment:
Same version as in `main`?

Reply:
We use `3.0.0-SNAPSHOT` in main-lts' pom.xml: https://github.com/quarkiverse/quarkus-openapi-generator/blob/287965b2d6557e33265072c2cb486abf0e455a64/pom.xml#L11
Is that a problem?